cmd/shfmt/docker-entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (C) 2019 Olliver Schinagl <oliver@schinagl.nl>
+#
+# A beginning user should be able to docker run image bash (or sh) without
+# needing to learn about --entrypoint
+# https://github.com/docker-library/official-images#consistency
+
+set -eu
+
+# run command if it is not starting with a "-" and is an executable in PATH
+if [ "${#}" -gt 0 ] && \
+   [ "${1#-}" = "${1}" ] && \
+   command -v "${1}"; then
```

Comment:
Shouldn't that add `>/dev/null 2>&1` to  avoid the output?

Reply:
> Shouldn't that add `>/dev/null 2>&1` to avoid the output?

Hah, you are right, I don't understand where I have lost it, as I just checked a different repo of mine, and sure enough > "/dev/null" 2>&1 right there.

So sorry, my bad!